Services/Interfaces/web_interface/templates/profile.html

```diff
@@ -119,19 +119,22 @@ <h4>Configuration issue</h4>
         {% endif %}
         <div class="tab-content" id="super-container">
             <div class="card card-header ">
-                <a class="d-inline-block" data-profile-id="{{current_profile.profile_id}}" data-role="current-profile-selector">
-                    <h2 class="">
-                        <span class="d-none d-md-inline-block">Current profile</span>
-                        <span class="font-weight-bold" data-role="profile-name" data-profile-id="{{current_profile.profile_id}}">{{current_profile.name}}</span>
-                    </h2>
-                </a>
-                <a class="btn btn-sm rounded-circle btn-primary waves-effect mx-1 mx-md-4 align-top"
-                   href="{{url_for('profiles_selector')}}" data-toggle="tooltip"
-                   title="Select another profile"
-                   id="profile-selector-link"
-                >
-                    <i class="fas fa-exchange-alt" aria-hidden="true"></i>
-                </a>
+                <h2>
+                    <span class="d-none d-md-inline-block">Current profile</span>
+                    <span class="font-weight-bold" data-role="profile-name" data-profile-id="{{current_profile.profile_id}}">{{current_profile.name}}</span>
+                    <a class="d-inline-block" data-profile-id="{{current_profile.profile_id}}" data-role="current-profile-selector">
+                        <button class="btn btn-sm btn-outline-primary align-middle">
+                            View profile
```

Comment:
<img width="1916" height="811" alt="image" src="https://github.com/user-attachments/assets/090a05ee-efca-40fb-9e79-1b9adaa102f4" />
this button will make it easier to see the profile page from which users can share their profile and import others